### ISSUE TITLE
Incognito mode / Clean usage traces

### ISSUE THREAD

**Issue by Wikinaut** [kiwix/kiwix-android#835]
In (inter alia) my pull request/s (in 2015) to the Wikipedia Android App https://phabricator.wikimedia.org/T116360 I proposed an "incognito mode" with the most important option to never store the browsing history. I never understood, why the Wikipedia Android App developers decided finally **not** to integrate my proposals.

I wish to propose this for KIWIX, too, and, if you agree, that this is useful, will commit myself to propose Pull Requests for the Kiwix-Android.

Basically, an "active incognito mode"

* clears the present history (when "icognito" gets activated); and
* will not store new search fragments ; and
* will not store browsed articles

**Comment by Wikinaut** [comment]
please let it open

**Comment by kelson42** [comment]
@Wikinaut We will keep it open, just need to be prioritise at some point next year.

**Comment by Wikinaut** [comment]
Stopp!

**Comment by macgills** [comment]
@Wikinaut you don't need to comment. We are keeping an eye on this, stalebot is just a bit of automation to help oragnisation of the project

>We will keep it open, just need to be prioritise at some point next year.

**Comment by Wikinaut** [comment]
Please keep it active.

**Comment by macgills** [comment]
@Wikinaut it is scheduled for a milestone, there is no need to comment to remove the stale tag

**Comment by KushagraPatni** [comment]
Hi,Please make the issue active.

**Comment by asmit27rai** [comment]
@kelson42 Pls Assign It To Me

**Comment by MohitMaliFtechiz** [comment]
@kelson42 This issue can be moved to the next milestone.
